### PR TITLE
typos fixed

### PR DESCRIPTION
**Pull request by Vivek-Prajapatii**
@Jeeppler typos fixed as you asked to do, if there is any other please let me know.

### CODE REVIEW

File: sechub-commons-pds/src/main/java/com/daimler/sechub/commons/pds/PDSConfigDataKeyProvider.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
      * 
      */
     PDS_SCAN_CONFIGURATION(new ExecutionPDSKey(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_CONFIGURATION,
-            "This conains the sechub storage location when sechub storage shall be used. So PDS knows location - in combination with sechub job UUID reuse is possible")
+            "This Contains the SecHub configuration as JSON object (but reduced to current scan type, so e.g. a web scan will have no code scan configuration data available")
```
Comment (Jeeppler) [inline sechub-commons-pds/src/main/java/com/daimler/sechub/commons/pds/PDSConfigDataKeyProvider.java:68]:
```suggestion
            "This contains the SecHub configuration as JSON object (but reduced to current scan type, so e.g. a web scan will have no code scan configuration data available")
```

**Review by Jeeppler — CHANGES_REQUESTED** [review]
Good job. However, I found one issue. Please improve this.

**Comment by Jeeppler** [comment]
@de-jcup and @sven-dmlr can you please explain what the extra requirements for a FOSS contribution at Mercedes-Benz are.

**Comment by Jeeppler** [comment]
@Vivek-Prajapatii to be able to accept your pull request we ask you to sign the Contributor License Agreement (CLA). Our Wiki page explains how to sign the Contributor License Agreement (CLA): https://github.com/Daimler/sechub/wiki/Collaboration.

In case you have questions regarding the CLA or the process, feel free to ask us.

**Review by Jeeppler — APPROVED** [review]
Good work.

**Comment by Jeeppler** [comment]
@de-jcup can you add your review as well?

**Review by de-jcup — CHANGES_REQUESTED** [review]
@Vivek-Prajapatii : Thank you for your contribution. Unfortunately I still have not your signed CLA . 

CLA signing is pretty simple - see https://github.com/Daimler/sechub/wiki/Collaboration#in-a-nutshell 

Please give me feedback when you have signed and mailed the CLA. After this I will approve and merge your PR.

**Comment by Vivek-Prajapatii** [comment]
Yes sure I'll sign it asap.

**Comment by Vivek-Prajapatii** [comment]
done with t he CLA @de-jcup

**Comment by de-jcup** [comment]
@Vivek-Prajapatii - thanks for contribution. 

Just a remark: You did not provide the issue number inside your commit messages, so it was not easy find the corresponding issue, assign the milestone and close it. But this was also a lack of information inside our wiki, so I just added these point there at https://github.com/Daimler/sechub/wiki/Collaboration#commit-messages. Please read and apply in future contributions.

**Comment by Jeeppler** [comment]
@Vivek-Prajapatii thank you for your contribution. We appreciate, that you took the effort to finish this pull-request by submitting the code and signing the Contributor License Agreement (CLA).

**Pull request merged**